src/views/Hearing/Sidebar.js

```diff
@@ -35,6 +37,27 @@ class Sidebar extends React.Component {
     );
   }
 
+  getLanguageChanger() {
+    const {hearing, activeLanguage, dispatch} = this.props;
+    const availableLanguages = { fi: 'Kuuleminen Suomeksi', sv: 'Frågorna tillgängliga', en: 'Questionaire in English'};
+    const languageOptionsArray = config.languages.map((lang) => {
+      if (getAttr(hearing.title, lang, {exact: true})) {
+        return (<div className={`language-link${lang === activeLanguage ? "-active" : ''}`}>
```

Comment:
Actually, I would display here as active the selected Hearing language, not the UI language. So the idea is that active indicates the language the Hearing is currently shown.